packages/commonwealth/server/CommonwealthConsumer/messageProcessors/snapshotConsumer.ts

```diff
@@ -0,0 +1,91 @@
+import axios from 'axios';
+import {factory, formatFilename} from 'common-common/src/logging';
+import {SnapshotNotification} from '../../../shared/types';
+import {DB} from "../../models";
+
+
+export async function processSnapshotMessage(this: { models: DB }, data: SnapshotNotification) {
+  const log = factory.getLogger(formatFilename(__filename));
```

Comment:
We should initialize the logger in the root namespace not in the function call.

Reply:
for rich